hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientExecutorServiceProxy.java

```diff
@@ -593,6 +593,7 @@ public void onResponse(T response) {
 
         @Override
         public void onFailure(Throwable t) {
+            multiExecutionCallbackWrapper.onResponse(member, t);
```

Comment:
I wonder whether this line should be backported to 3.9.x ?

Reply:
Yes, this would be good. It's a back in error propagation.

Reply:
@jerrinot https://github.com/hazelcast/hazelcast/pull/12075